agent/task.go

```diff
@@ -1,12 +1,12 @@
 package agent
 
 import (
-	"reflect"
 	"time"
 
 	"github.com/docker/swarmkit/agent/exec"
 	"github.com/docker/swarmkit/api"
 	"github.com/docker/swarmkit/log"
+	"github.com/docker/swarmkit/manager/dispatcher"
```

Comment:
The agent should not import the dispatcher. If there is common code, we should put it in a shared package.